applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -781,16 +839,16 @@ static void disconnected_cb(struct bt_conn *conn, uint8_t reason)
 static int discover_sink(struct bt_conn *conn)
 {
 	int ret = 0;
-	static uint8_t conn_index;
+	static uint8_t discover_index;
```

Comment:
I see some potential issues with having this static. Let's discuss.

Reply:
This needs to be static so that we don't try to discover two sinks at the same time with the same discover parameters